defs/scenarios/openstack/keystone/bugs.yaml

```diff
@@ -11,6 +11,14 @@ checks:
         # Victoria
         - min: 2:18.0.0
           max: 2:18.0.0-99
+    config:
+      handler: hotsos.core.plugins.openstack.OpenstackConfig
+      path: etc/keystone/keystone.conf
```

Comment:
The [ldap] option can be turned on per-domain basis [1], so if we want this to work properly we also need to check all .conf files under `/etc/keystone/domains/` as well.

[1] See `Additional LDAP integration settings` section at https://docs.openstack.org/keystone/zed/admin/configuration.html#identity-ldap-server-set-up

Reply:
The domain names can be arbitrary so I'm unsure if doing this is possible right now, we would need to allow wildcards in the path.

Reply:
Good catch! I have now added support to do this and updated the patch.

Reply:
ugh just realised my domain config path is not valid, ill fix that now

Reply:
done.